schema/object-upload.yml

```diff
@@ -0,0 +1,181 @@
+paths:
+  /v1/object-upload:
+    get:
+      summary: List object-uploads
+      operationId: getObjectUploads
+      description: By passing in query string, user can get available object-uploads authorised
+      parameters:
+        - in: query
+          name: filter[rsql]
+          description: pass an optional search string for looking up the object-uploads
+          schema:
+            type: string
+        - in: query
+          name: sort
+          description: optional sort string, can have sort order such as descending denoted by "-"
+          schema:
+            type: string
+        - in: query
+          name: page[offset]
+          description: number of records to skip when paging
+          schema:
+            type: integer
+            format: int32
+        - in: query
+          name: page[limit]
+          description: maximum number of records to return when paging
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+            maximum: 50
+      responses:
+        '200':
+          description: object-uploads satifying the query restrictions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ObjectUpload'
+        '400':
+          description: bad input parameter
+        '404':
+          description: object-uploads not found
+  /v1/object-upload/{Id}:
+    get:
+      summary: Get an object-upload
+      description: Return a single object-upload
+      operationId: getObjectUploadById
+      parameters:
+        - name: Id
+          in: path
+          description: ID of object-upload to return
+          required: true
+          schema:
+            type: string
+            format: uuid            
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ObjectUpload'
+        "400":
+          description: Invalid ID supplied
+        "404":
+          description: object-upload not found
+  /v1/file/{bucket}:        
+    post:
+      summary: Upload a file to minio bucket
+      operationId: uploadFile
+      description: Upload a file to minio bucket
+      responses:
```

Comment:
We need to define the bucket in the parameters

```
parameters:
        - name: bucket
          in: path
          description: bucket of the file
          required: true
          schema:
            type: string
```

Reply:
If you look at here : https://github.com/DINA-Web/object-store-specs/blob/Support--21095--Create-OpenAPI-3-file-for-ObjectUpload/schema/object-upload.yml#L85
The parameter is described in request body of form-data